janusgraph-core/src/main/java/org/janusgraph/graphdb/database/serialize/attribute/UUIDSerializer.java

```diff
@@ -33,12 +34,6 @@ public UUID read(ScanBuffer buffer) {
         return new UUID(mostSignificantBits, leastSignificantBits);
     }
 
-    @Override
```

Comment:
minor, it seems like the content of the write() method did not change. so, can you keep it in the same place to reduce the number of diffs in this PR?

Reply:
@acmp Sure thing. Fixed,